README.md

```diff
@@ -54,6 +54,14 @@ crate features, WASM support and Custom RNGs see the
 
 This crate requires Rust 1.36.0 or later.
 
+## Platform Support
+
+This crate supports the same operating system or platform versions that the Rust standard library does, unless a
+[custom implementation](https://docs.rs/getrandom/latest/getrandom/#custom-implementations) is used instead.
```

Comment:
The "unless" part looks a bit unclear to me. Currently we do not allow custom backends to overwrite support for built-in targets. I think it could be better to write it like this:

>This crate generally supports the same operating system and platform versions that the Rust standard library does. Additional targets may be supported using plugable custom implementations.

Reply:
That sounds better to be as well, thanks. Applied it and copy/pasted into the crate documentation.